verticapy/tests_new/performance/vertica/collection/test_export_simple.py

```diff
@@ -0,0 +1,194 @@
+permissions and limitations under the License.
+"""
+import logging
+import os
+import tarfile
+
+import pytest
+
+from verticapy.performance.vertica import QueryProfiler
+from verticapy.performance.vertica.collection.profile_import import (
+    ProfileImport,
+)
+from verticapy._utils._sql._sys import _executeSQL
+
+
+class TestQueryProfilerSimple:
+    """
+    Contains tests related to creating bundles of parquet data
+    """
+
+    def test_profile_export(self, amazon_vd, schema_loader, tmp_path):
+        logging.info(f"Amazon vd is {amazon_vd}")
+        request = f"""
+        SELECT 
+            date, 
+            MONTH(date) AS month, 
+            AVG(number) AS avg_number 
+        FROM 
+            {amazon_vd}
+        GROUP BY 1;
+        """
+
+        qp = QueryProfiler(request, target_schema=schema_loader)
+
+        outfile = tmp_path / "qprof_test_001.tar"
+        logging.info(f"Writing to file: {outfile}")
+        qp.export_profile(filename=outfile)
+        logging.info(
+            f"Files in tmpdir: {','.join([str(x) for x in tmp_path.iterdir()])}"
+        )
+        assert os.path.exists(outfile)
+
+        tf = tarfile.open(outfile)
+
+        tarfile_contents = set(tf.getnames())
+
+        expected_files = set(
+            [
+                "dc_explain_plans.parquet",
+                "dc_query_executions.parquet",
+                "dc_requests_issued.parquet",
+                "execution_engine_profiles.parquet",
+                "host_resources.parquet",
+                "query_consumption.parquet",
+                "query_plan_profiles.parquet",
+                "query_profiles.parquet",
+                "resource_pool_status.parquet",
+                "profile_metadata.json",
+            ]
+        )
+
+        # Recall: symmetric difference is all elements that are in just
+        # one set.
+        set_diff = tarfile_contents.symmetric_difference(expected_files)
+
+        assert len(set_diff) == 0
+
+    def test_profile_full_lifecycle(self, amazon_vd, schema_loader, tmp_path):
+        request = f"""
+        SELECT 
+            date, 
+            MONTH(date) AS month, 
+            AVG(number) AS avg_number 
+        FROM 
+            {amazon_vd}
+        GROUP BY 1;
+        """
+
+        qp = QueryProfiler(request, target_schema=schema_loader)
+
+        outfile = tmp_path / "qprof_test_002.tar"
+        logging.info(f"Writing to file: {outfile}")
+        qp.export_profile(filename=outfile)
+        assert os.path.exists(outfile)
+
+        key = "reload123"
+        pi = ProfileImport(
+            target_schema=schema_loader,
+            key=key,
+            filename=outfile,
+        )
+        unpack_tmp = tmp_path / "unpack"
+        unpack_tmp.mkdir()
+        pi.tmp_path = unpack_tmp
+        pi.check_schema_and_load_file()
+
+        loaded_tables = self._get_set_of_tables_in_schema(schema_loader, key)
+
+        expected_tables = [
```

Comment:
can we define this outside class as this is getting used multiple times?

Reply:
removed the entire test where this was repeated... the test was a duplicate